docs/getting-started/deployment-other.md

```diff
@@ -33,3 +33,21 @@ $ heroku container:release web -a <your-app>
 ```
 
 With that, you should have Backstage up and running!
+
+## AWS Fargate and Aurora
+
+There are any number of ways to deploy backstage containers on AWS. One of the
+simplest from a management and ops perspective is to leverage ECS Fargate and
+Aurora PostgreSQL, where both the container orchestration and database clusters
+are AWS managed services.
+
+However, to use them properly there are a large number of supporting resources
+(VPCs, Security Groups, Load Balancers, Certificate, etc) required. One approach is to use
+[AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/) to simplify and
+automate the creation of the entire infrastructure stack along with automating
+the build and deploy of the container.
+
+Check out this repo:
+[Backstage On AWS](https://github.com/rbogle/backstage-on-aws), for a ready to
```

Comment:
Looping back to the discussion about moving this to contrib. Rather than linking directly to the repo here, could we link to a README in contrib that then in turn links out to the external repo? Wrapped up in just some tiny description. Not too happy about linking to external repos straight from these docs as we don't really know what state they're in, how up to date they are, etc.

Reply:
Yes, that makes more sense: keeping the official docs decoupled from any external links. 
Shall I close this PR, and submit a new one with this change pointing to contrib/fargate and a readme there?

Reply:
It's fine to add those changes to this PR if that's alright

Reply:
Yep will do! Thanks for the feedback.